build.gradle.kts

```diff
@@ -108,6 +109,7 @@ kotlin {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
+                implementation("org.mock-server:mockserver-netty:$mockServerVersion")
             }
```

Comment:
**Nit**: Discussed offline, MockServer belongs in **jvmTest** since it's not a KMP client.

Reply:
Correct that this dependency is jvm only. If we add this we basically have to move the tests to **jvmTest** as well and we won't be able to have common test suite anymore. This is _ok_ for right now but eventually we'll be wiring up a kotlin/native version as well.

Reply:
I relocated them for now, we can discuss if that's the best thing long-term